Sources/ElasticSwift/Requests/SearchRequest.swift

```diff
@@ -179,30 +161,18 @@ public class SearchRequest<T: Codable>: Request {
         return try JSONSerialization.data(withJSONObject: dic, options: [])
     }
     
-    func responseHandler(_ response: ESResponse) -> Void {
-        if let error = response.error {
-            return completionHandler(nil, error)
-        }
-        do {
-            debugPrint(String(data: response.data!, encoding: .utf8)!)
-            let decoded: SearchResponse<T>? = try Serializers.decode(data: response.data!)
-            if decoded?.took != nil {
```

Comment:
Is not anymore needed to check for mandatory NIL fields on response deserialization because the target classes have now mandatory fields nonOptionals and if the ESResponse doesn't contains them (in case of error), the Deserializer throws an error.
Then Error responses are managed using Deserialized errors and there is no need to check NIL manually for each different key attribute in different responses.

See 
```
extension Response {
    
    public static func create(fromESResponse response: ESResponse, withSerializer serializer: Serializer) throws -> Self {
...
   }
}
```